aioauth/collections.py

```diff
@@ -9,6 +9,7 @@
 """
 
 from collections import UserDict
+from typing import Any
 
 
 class HTTPHeaderDict(UserDict):
```

Comment:
To "solidify" these changes, I would create a test file for collections.py (e.g., test_collections.py) with tests like these:

```python
def test_http_header_dict_case_insensitive_get():
    headers = HTTPHeaderDict(Authorization="Bearer token")
    assert headers.get("authorization") == "Bearer token"
    assert headers.get("Authorization") == "Bearer token"
    assert headers.get("AUTHORIZATION") == "Bearer token"
    assert headers.get("nonexistent", "default") == "default"


def test_http_header_dict_case_insensitive_delitem():
    headers = HTTPHeaderDict(Authorization="Bearer token")
    del headers["authorization"]
    assert headers.get("Authorization") is None


def test_http_header_dict_case_insensitive_init():
    headers = HTTPHeaderDict(**{"Authorization": "Bearer token", "Content-Type": "application/json"})
    assert headers.get("authorization") == "Bearer token"
    assert headers.get("content-type") == "application/json"
```